app/src/main/kotlin/net/syncthing/lite/utils/MimeType.kt

```diff
@@ -12,6 +12,6 @@ object MimeType {
     }
 
     fun getFromUrl(url: String) = getFromExtension(
-            MimeTypeMap.getFileExtensionFromUrl(url)
+            MimeTypeMap.getFileExtensionFromUrl(url).toLowerCase()
```

Comment:
Shouldn't you do toLowerCase() before getting the mime type? And do the same in getFromExtension()?

Reply:
It is applied to the file extension before the file extension is passed to getFromExtension()

Reply:
Oh you're right, I misread the code. Looks good actually!